requirements.txt

```diff
@@ -23,6 +23,7 @@ django-timezone-field==4.0
 djangorestframework==3.11.0
 exodus-core==1.3.1
 future==0.17.1
+google-play-scraper==0.1.2
 gpapi==0.4.4
 GPlayCli==3.26
```

Comment:
Could you please remove this dependency since afaik it is not required anymore.

Reply:
Done